website/docs/usage/index.mdx

```diff
@@ -259,6 +259,26 @@ source code and recompiling frequently.
   $ python setup.py develop
   ```
 
+#### Additional tooling
+
```

Comment:
It'd be nice to add one of the gifs or screenshots here because visuals communicate much better how cool the extension is ✨

Reply:
I couldn't figure out how to add it! Was going to comment. Is there a guide on how?

Reply:
You can add images to `website/public/images/`